pubspec.yaml

```
@@ -59,32 +59,20 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
-  #   - images/a_dot_ham.jpeg
+  assets:
+    - assets/logo_square.png
+    - assets/intro.gif
```

수진님 다음과 같이 변경하면 폴더안에 있는 이미지들을 일일이 명명해주지않아도 다 사용할 수 있더라구요!
assets 폴더 안에 images 폴더 만들어주시면 좋을 것 같습니다!
```suggestion
  assets:
    - assets/images/
```

---

확인했습니다!